azext_edge/edge/providers/orchestration/work.py

```diff
@@ -574,7 +574,7 @@ def render_display(self, category: Optional[WorkCategoryKey] = None, active_step
     def stop_display(self):
         if self._render_progress and self._live.is_started:
             if self._progress_shown:
-                self._progress_bar.update(self._task_id, description="DONE!")
+                self._progress_bar.update(self._task_id, description="Done.")
```

Comment:
i like DONE! more

Reply:
Currently DONE! is shown even in case of errors, which may convey a happy state even though it's not. We can make a different output for errors, but I don't want to do it in this PR.